galette/webroot/installer.php

```diff
@@ -202,12 +202,19 @@
     $install->atEndStep();
 }
 
+$documentation_language = $i18n->getAbbrev();
```

Comment:
A similar condition to fix locales names has also been added. I guess a method managed from the I18n object would be preferable (it should ba available on both cases).

Reply:
All right!